deployment/modules/github/org/repositories.tf

```diff
@@ -0,0 +1,139 @@
+variable "repositories" {
+  type = list(object({
+    name        = string
+    description = string
+    discussions = optional(bool)
+    projects    = optional(bool)
+    issues      = optional(bool)
+    archived    = optional(bool)
+  }))
+  default = [
+    {
+      name        = "immich", description = "High performance self-hosted photo and video management solution.",
+      discussions = true, projects = true
+    },
+    { name = "devtools", description = "Various tooling used by the Immich maintainer team" },
+    {
+      name = "my.immich.app", description = "Redirect urls to personal, hosted, instances of Immich.",
+    },
+    { name = "base-images", description = "Base images for Immich containers" },
+    { name = "immich-charts", description = "Helm chart implementation of Immich" },
+    { name = "discord-bot", description = "A Discord bot for the official @immich-app Discord" },
+    { name = "demo", description = "This repo contains the setup for the demo instance at https://demo.immich.app/" },
+    { name = "test-assets", description = "Test assets used for testing Immich. Contains various formats and codecs" },
+    { name = ".github", description = ".github folder for the organisation level", issues = false },
+    {
+      name        = "CLI", archived = true,
+      description = "CLI utilities for Immich to help with upload images and videos from a location on a desktop machine or a server to the Immich's server",
+    }
+  ]
+}
+
+resource "github_repository" "repositories" {
+  for_each                  = { for repo in var.repositories : repo.name => repo }
+  name                      = each.value.name
+  description               = each.value.description
+  allow_auto_merge          = true
+  allow_merge_commit        = false
+  allow_rebase_merge        = false
+  allow_squash_merge        = true
+  allow_update_branch       = true
+  archived                  = false
```

Comment:
This'll override cli from the list above

```
  # github_repository.repositories["CLI"] will be updated in-place
  # (imported from "CLI")
  ~ resource "github_repository" "repositories" {
      (...)
      ~ archived                    = true -> false
```

Reply:
Fixed